app/views/providers/confirm_dwp_non_passported_applications/show.html.erb

```diff
@@ -1,25 +1,33 @@
-<% error_class = @error ? 'alternative-error-style' : '' %>
+<% error_class = @form.errors.any? ? 'alternative-error-style' : '' %>
 
```

Comment:
this could be done in a separate PR but I think a better way of doing this would be to just override the error styles in the scss file for the interruption panel class - since the error styling will always be different for these panels

Reply:
I added that to fix the form errors which the additions I made. But yeah I could stick on ID or class or something to do it instead.